joplin/base/models/janis_page.py

```diff
@@ -42,6 +42,8 @@ class JanisBasePage(Page):
         verbose_name='Notes for authors (Not visible on the resident facing site)'
     )
 
+    coa_global = models.BooleanField(default=False, verbose_name='Make this a top level page')
```

Comment:
i think the verbose name needs to be updated (and im still sad we're abusing verbose name so much)

Reply:
That's one of the things I talked about needing to review with design/content.

Reply:
since we're just trying to get the privacy policy up asap, I've created https://github.com/cityofaustin/techstack/issues/3051 to handle this